core/src/main/resources/db/changelog/changesets/2024_05_08_dsm_env_config.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: "dsm_env_config"
+      author: dbush
+      changes:
+        - addColumn:
+            tableName: study_environment_config
+            columns:
+              - column: { name: use_dev_dsm_realm, type: boolean, defaultValueBoolean: true, constraints: { nullable: false} }
+              - column: { name: use_stub_dsm, type: boolean, defaultValueBoolean: true, constraints: { nullable: false} }
+        - addDefaultValue:
+            tableName: study_environment_config
+            columnName: accepting_proxy_enrollment
+            defaultValueBoolean: false
+        - sql:
```

Comment:
make sure we auto-update ourhealth live to use real kits so we don't forget to do that in product -- everything else we can set manually.